pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -477,10 +477,14 @@ class Build {
                         def rerunIterations = '3'
                         def fipsTestBuildSuffix = "";
                         def buildList = ""
+                        // testType value examples: 
+                        // extended.functional.fips140_3_OpenJCEPlusFIPS.FIPS140-3
+                        // sanity.jck.fips140_3_OpenJCEPlusFIPS
```

Comment:
Apologies for my ignorance on this topic but why do we have `fips140_3` as well as `.FIPS140-3`? Seems redundant.

Reply:
`fips140_3` was used to follow the same format as `fips140_2`. `OpenJCEPlusFIPS.FIPS140-3` is the real FIPS profile value and is defined by the FIPS team. It will become a JVM option (i.e., `-Dsemeru.customprofile=OpenJCEPlusFIPS.FIPS140-3`). In short, we can say `fips140_xxx` is for the test pipeline and the value after it is for JVM options.

It is not really redundant as we have 2 FIPS140-3 profiles :
- fips140_3_OpenJCEPlusFIPS (weak profile)
- fips140_3_OpenJCEPlusFIPS.FIPS140-3 (strict profile)

We also have `fips140_2` (FIPS140-2 profile). And we will add `openjceplus` (non-FIPS openjceplus) soon.